### PR TITLE
[BUGFIX] Use more precise extension key

### PR DESCRIPTION
**Pull request by liayn**
To avoid name clashes we use a more precise extension key.

Resolves: #8

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "extra": {
         "typo3/cms": {
             "cms-package-dir": "{$vendor-dir}/typo3/cms",
-            "extension-key": "skilldisplay",
+            "extension-key": "skilldisplay-content",
```
Comment (DanielSiepmann) [inline composer.json:58]: That key is not valid, cited from docs:
> 2 .The extkey MUST be made up of lowercase alphanumeric characters and underscores only and MUST start with a letter.
>  —https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ExtensionArchitecture/NamingConventions/Index.html#extension-key-extkey

```suggestion
            "extension-key": "skilldisplay_content",
```

Comment (liayn) [reply composer.json:58]: args... totally right. no need to cite the docs, I was just "stupid". Will fix asap

**Comment by liayn** [comment]
I'll also change the namespace of the classes to match the new extension key

**Comment by Weissheiten** [comment]
I required the branch for this fix on my testing instance:

- extension key is correct
- the database tool offers the possibility to add the required atrributes to the table

Problem:
The Skills Tab does not appear any more when adding new content elements, therefore the content element cannot be selected/added

**Comment by liayn** [comment]
Oh dear... fixed a lot more files now, where the extkey is part of. Please try again.

File: composer.json [hunk 2]
```diff
@@ -55,7 +55,7 @@
     "extra": {
         "typo3/cms": {
             "cms-package-dir": "{$vendor-dir}/typo3/cms",
-            "extension-key": "skilldisplay",
+            "extension-key": "skilldisplay_content",
```
Comment (DanielSiepmann) [inline composer.json:58]: Also needs to be changed in line 52, where it is symlinked for functional tests.
We still don't have functionals, but once an functional test is added, the extension name would be different.

**Review by DanielSiepmann — CHANGES_REQUESTED** [review]
When changing extension key, also extension name has to be changed.
E.g. within `Resources/Private/Partials/ContentElements/VerificationBox.html`, usage on line 25: `extensionName: 'skilldisplay',`

`Configuration/SiteConfiguration/Overrides/sites.php` still uses the old extension key in the last line.

File: composer.json (hunk 2 above)
Comment (liayn) [reply composer.json:58]: Right and actually that's even outdated, will replace with: `TYPO3\\TestingFramework\\Composer\\ExtensionTestEnvironment::prepare`

**Review by DanielSiepmann — APPROVED** [review]

**Pull request merged**